src/main/java/models/Game.java

```diff
@@ -87,6 +87,20 @@ else if (dealerScore != userScore)
 			userWon = true;
 	}
 
+    // A player choosing to stay is essentially a signal
+    // for the dealer's turn to start
+    public void stay(){
+        dealCardToDealer(); // Call twice at turn start
+        dealCardToDealer();
+        // Do we need to turn one of these cards face down? Or will that
```

Comment:
This actually needs to be done at the start of the game. The user gets to see what one of the two cards the dealer has while they are playing. I can just do that from the deal function in the controller I think.